src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -35,11 +37,30 @@ const generateInitialOptions = series => {
             ).toFixed(2);
 
             tooltipData.push(`Δ ${delta} (${deltaPercentage}%)`);
+
+            const currRevision = series.data[tooltipItems[0].index].revision;
+            const prevRevision =
+              series.data[tooltipItems[0].index - 1].revision;
+            const hgLink = `https://hg.mozilla.org/mozilla-central/pushloghtml?fromchange=${prevRevision}&tochange=${currRevision}`;
+
+            tooltipData.push(
+              `<a href="${hgLink}" target="_blank">${currRevision.slice(
```

Comment:
instead of this format string, use JSX

              <a href={hgLink} target="_blank">{currRevision.slice(0,12)}</a>